staging/src/k8s.io/apiserver/pkg/apis/apiserver/types.go

```diff
@@ -150,6 +150,7 @@ type TLSConfig struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // TracingConfiguration provides versioned configuration for tracing clients.
+// This type will go away in favor of component-base/config TracingConfiguration
```

Comment:
Is this intended to be a deprecation notice?

```suggestion
// This type is deprecated in favor of component-base/config TracingConfiguration
```